BlueCombing/Views/MakeCardViews/BadgeDescription.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  BadgeDescription.swift
+//  BlueCombing
+//
+//  Created by ryu hyunsun on 2022/10/15.
+//
+
+import UIKit
+
+extension String {
+    func englishBadge() -> BadgeDescription {
+        switch self {
+        case "산호초":
+            return .coral
+        case "조개":
+            return .clam
+        case "해마":
+            return .hippocampus
+        case "해파리":
+            return .jellyfish
+        case "돌고래":
+            return .dolphin
+        case "거북이":
+            return .turtle
+        case "물고기 떼":
+            return .fishs
+        case "고래":
+            return .whale
+        default:
+            return .coral
+        }
+
+    }
+}
+
+
+enum BadgeDescription {
```

Comment:
```suggestion
enum BadgeDescription: String {
```
어떤 반환형 인지 써주면 좋을 것 같습니다.

Reply:
딱히 반환값이 필요 없어, 타입을 쓰지 않았었습니다! 조언해주신대로 반환형 추가하여 커밋했습니다.